config.md

```diff
@@ -328,7 +328,7 @@ Hooks allow for the configuration of custom actions related to the [lifecycle](r
     Entries in the array contain the following properties:
     * **`path`** (string, REQUIRED) with similar semantics to [IEEE Std 1003.1-2001 `execv`'s *path*][ieee-1003.1-2001-xsh-exec].
       This specification extends the IEEE standard in that **`path`** MUST be absolute.
-    * **`args`** (array of strings, REQUIRED) with the same semantics as [IEEE Std 1003.1-2001 `execv`'s *argv*][ieee-1003.1-2001-xsh-exec].
+    * **`args`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2001 `execv`'s *argv*][ieee-1003.1-2001-xsh-exec].
```

Comment:
If we're thinking about adjusting this, I [still][1] prefer Python's approach which makes `path` optional.  I'd also like to [unify the approach to process invocation between `hooks` and `process`][2] (having different semantics for `args` depending on the context is just confusing).  But if those are not on the table, this change looks ok to me, and gets us back closer to the [old][3] [Go semantics with their optional Args][4].

[1]: https://groups.google.com/a/opencontainers.org/d/msg/dev/luaaorsya10/IdjzUYl9GQAJ
[2]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/Cn5L6g1prgA
[3]: https://github.com/opencontainers/runtime-spec/pull/427/files#diff-c9c91c29b41257aea3a3403cc606ad99L383
[4]: https://golang.org/pkg/os/exec/#Cmd